src/models/ThemeProvider.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 import React, { Component, type Element } from 'react'
 import PropTypes from 'prop-types'
-import isPlainObject from 'is-plain-object'
```

Comment:
Can this dependency be removed entirely now?

Reply:
It's also used in `utils/flatten.js` to determine whether to try flattening the object or to just call `toString` on it. I think in that case it's still necessary that the object be "plain."